class.json-api-endpoints.php

```diff
@@ -1139,7 +1141,8 @@ function get_author( $author, $show_email = false ) {
 			$avatar_URL = $this->api->get_avatar_url( $email );
 		}
 
-		$email = $show_email ? (string) $email : false;
+		$email = $show_email_and_ip ? (string) $email : false;
+		$ip_address = $show_email_and_ip ? (string) $ip_address : false;
```

Comment:
I didn't want to change the existing `string|false` behaviour for `$email`, and I just reused it for `$ip_address` since it's shown for the same context/permissions.